panda/src/wgldisplay/wglGraphicsStateGuardian.cxx

```diff
@@ -611,8 +611,10 @@ make_context(HDC hdc) {
       attrib_list[n++] = WGL_CONTEXT_DEBUG_BIT_ARB;
     }
 #ifndef SUPPORT_FIXED_FUNCTION
-    attrib_list[n++] = WGL_CONTEXT_PROFILE_MASK_ARB;
-    attrib_list[n++] = WGL_CONTEXT_CORE_PROFILE_BIT_ARB;
+    if(!has_fixed_function_pipeline()) {
+      attrib_list[n++] = WGL_CONTEXT_PROFILE_MASK_ARB;
+      attrib_list[n++] = WGL_CONTEXT_CORE_PROFILE_BIT_ARB;
+    }
```

Comment:
I think this `if` check shouldn't be here.  This is in the context creation code, so `_core_profile` hasn't been initialised yet.  Instead, I think we should check if gl_version is 3.2 or higher.